From 78976a2b547d388b77fef369cca872ab879beb88 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Wed, 25 Feb 2009 17:29:34 -0800
Subject: Staging: slicoss: remove DEBUG_REGISTER_TRACE

It's not ever defined, so remove it from the code base.

Cc: Lior Dotan <liodot@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/slicoss/slic_os.h |   27 ---------------------------
 drivers/staging/slicoss/slicoss.c |   15 ---------------
 2 files changed, 42 deletions(-)

--- a/drivers/staging/slicoss/slic_os.h
+++ b/drivers/staging/slicoss/slic_os.h
@@ -42,37 +42,10 @@
 #ifndef _SLIC_OS_SPECIFIC_H_
 #define _SLIC_OS_SPECIFIC_H_
 
-#ifdef DEBUG_REGISTER_TRACE
-#define WRITE_REG(reg, value, flush)                                      \
-	{                                                           \
-		adapter->card->reg_type[adapter->card->debug_ix] = 0;   \
-		adapter->card->reg_offset[adapter->card->debug_ix] = \
-		((unsigned char *)(&reg)) - \
-		((unsigned char *)adapter->slic_regs); \
-		adapter->card->reg_value[adapter->card->debug_ix++] = value;  \
-		if (adapter->card->debug_ix == 32) \
-			adapter->card->debug_ix = 0;                      \
-		slic_reg32_write((&reg), (value), (flush));            \
-	}
-#define WRITE_REG64(a, reg, value, regh, valh, flush)                        \
-	{                                                           \
-		adapter->card->reg_type[adapter->card->debug_ix] = 1;        \
-		adapter->card->reg_offset[adapter->card->debug_ix] = \
-		((unsigned char *)(&reg)) - \
-		((unsigned char *)adapter->slic_regs); \
-		adapter->card->reg_value[adapter->card->debug_ix] = value;   \
-		adapter->card->reg_valueh[adapter->card->debug_ix++] = valh;  \
-		if (adapter->card->debug_ix == 32) \
-			adapter->card->debug_ix = 0;                      \
-		slic_reg64_write((a), (&reg), (value), (&regh), (valh), \
-				(flush));\
-	}
-#else
 #define WRITE_REG(reg, value, flush) \
 	slic_reg32_write((&reg), (value), (flush))
 #define WRITE_REG64(a, reg, value, regh, valh, flush) \
 	slic_reg64_write((a), (&reg), (value), (&regh), (valh), (flush))
-#endif
 
 #endif  /* _SLIC_OS_SPECIFIC_H_  */
 
--- a/drivers/staging/slicoss/slicoss.c
+++ b/drivers/staging/slicoss/slicoss.c
@@ -245,21 +245,6 @@ static void slic_dbg_macaddrs(struct ada
 	return;
 }
 
-#ifdef DEBUG_REGISTER_TRACE
-static void slic_dbg_register_trace(struct adapter *adapter,
-					struct sliccard *card)
-{
-	uint i;
-
-	DBG_ERROR("Dump Register Write Trace: curr_ix == %d\n", card->debug_ix);
-	for (i = 0; i < 32; i++) {
-		DBG_ERROR("%2d %d %4x %x %x\n",
-			  i, card->reg_type[i], card->reg_offset[i],
-			  card->reg_value[i], card->reg_valueh[i]);
-	}
-}
-#endif
-
 static void slic_init_adapter(struct net_device *netdev,
 			      struct pci_dev *pcidev,
 			      const struct pci_device_id *pci_tbl_entry,
